examples/onpremise/metadataConsole.php

```diff
@@ -46,73 +46,10 @@
  * - 51degrees/fiftyone.devicedetection
  */
 
-require_once(__DIR__ . "/exampleUtils.php");
 require_once(__DIR__ . "/../../vendor/autoload.php");
```

Comment:
also this one?

Reply:
Same as the other examples. They're not part of the package which would be used by client code, so they're not covered by the autoloader.